### ISSUE TITLE
Update values.toString to output brackets

### ISSUE THREAD

**Issue by bkutil** [juttle/juttle#177]
Based on the outcome of the discussion on gitter about `toString` output of arrays/objects, either update the code or close this issue.

**Comment by bkutil** [comment]
As discussed on gitter, `values.inspect` and `values.toString` should be modified so that

```
const c1 = null;
const c2 = 5;
const c3 = "abcd";
const c4 =  :2016-01-01 12:34:56:;

const s = "Interpolated: ${c1} ${c2} ${c3} ${c4}";
```
should interpolate to:

```
"Interpolated: null 5 abcd 2016-01-01T01:01T12:34:56.000Z"
```

and:

```
const c1 = [ null, 5, "abcd",  :2016-01-01 12:34:56: ];
const c2 = { a: null, b: 5, c: "abcd", d: :2016-01-01 12:34:56: };
const c3 = [];
const c4 = {};

const s = "Interpolated: ${c1} ${c2} ${c3} ${c4}";
```

to:

```
"Interpolated: [ null, 5, "abcd", "2016-01-01T01:01T12:34:56.000Z" ] { a: null, b: 5, c: "abcd", d: "2016-01-01T01:01T12:34:56.000Z" } [] {}"
```

**Comment by bkutil** [comment]
At the moment, we seem to have 3 different ways how to accomplish the same task - serializing a point into a string:

Je trochu nejasny, jak postupovat ohledne vystupu nested points:
1. `toString`
2. `inspect` - if we implement the above proposal from Rodney for `toString`, `inspect` can be dropped
3. `fromNative` - Mike suggested to keep the output structure of the point 'flat' - basically just keys + JSON for nested fields. This looks like a duplicate, too.

I'd propose folding those three implementations into a single one - `toString()`, which has:

1. human-readable output for flat and nested fields
2. represents the value uniquely, so it can be used as a group identifier

**Comment by bkutil** [comment]
This was codified in a spec https://github.com/juttle/juttle/wiki/String-Representation-of-Juttle-Values and implemented in #200
